src/idb.filesystem.js

```diff
@@ -70,7 +71,10 @@ const support = function() {
   };
 };
 
-const Base64ToBlob = function(dataURL) {
+// this is called to set IDB_SUPPORTS_BLOB at all times
+support();
```

Comment:
The was my bad. Let's rename the function to `setSupportsBlob()` and call it here.